### PR TITLE
Create package.json

### PR DESCRIPTION
**Pull request by r-brown**
Adding GuideChimp library ( Source: https://github.com/Labs64/GuideChimp )

# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by r-brown** [comment]
> Some keys are not needed in the package.json, it might be documented. We also need to bring pas the linting in CI.

Hi @xtuc,
- unnecessary tags deleted
- can you be more specific here please _"bring pas the linting in CI"_

**Comment by MattIPv4** [comment]
Currently, there is no actual CI that is running. We need to renable the CI to lint package.json files so that our manual reviews aren't needed on everything. Not specific to your PR, just a general issue.

**Comment by r-brown** [comment]
> Currently, there is no actual CI that is running. We need to renable the CI to lint package.json files so that our manual reviews aren't needed on everything. Not specific to your PR, just a general issue.

Yes, it is - original repo using GitHub Actions as CI - https://github.com/Labs64/GuideChimp/actions

**Comment by MattIPv4** [comment]
CI here, not your repository. "Not specific to your PR, just a general issue."

**Comment by r-brown** [comment]
> CI here, not your repository. "Not specific to your PR, just a general issue."

Hi @MattIPv4 
So what you are saying - Pull Requests are not possible in the cdnjs at the moment, as there is no CI support on your side.

Shall we close this PR then, as I do not see a way, how this can be resolved on our side?

File: ajax/libs/guidechimp/package.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            "name": "Rudkovskiy Viascheslav",
+            "email": "xraskolx@gmail.com",
+            "url": "https://github.com/v-rudkovskiy"
+        }
+    ],
+    "main": "dist/guidechimp.js",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

File: ajax/libs/guidechimp/package.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            "url": "https://github.com/Labs64"
+        },
+        {
+            "name": "Rudkovskiy Viascheslav",
+            "email": "xraskolx@gmail.com",
+            "url": "https://github.com/v-rudkovskiy"
+        }
+    ],
+    "main": "dist/guidechimp.js",
+    "files": [
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

File: ajax/libs/guidechimp/package.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    ],
+    "scripts": {
+        "build": "node build/build.js",
+        "release": "npm run build && npm run test",
+        "dev": "webpack --progress --colors --watch --config build/webpack.dev.conf.js",
+        "test": "karma start test/karma.conf.js --single-run",
+        "test-for-travis": "karma start test/karma.conf.js --single-run --browsers Firefox",
+        "lint": "eslint --ext .js,.vue src test"
+    },
+    "dependencies": {
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

File: ajax/libs/guidechimp/package.json [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    },
+    "devDependencies": {
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

File: ajax/libs/guidechimp/package.json [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        "ora": "^4.0.3",
+        "rimraf": "^3.0.1",
+        "sass-loader": "^8.0.2",
+        "style-loader": "^1.1.3",
+        "uglifyjs-webpack-plugin": "^2.2.0",
+        "webpack": "^4.39.3",
+        "webpack-cli": "^3.3.7",
+        "webpack-merge": "^4.2.2"
+    },
+    "engines": {
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

File: ajax/libs/guidechimp/package.json [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        "node": ">= 8.0.0",
+        "npm": ">= 6.0.0"
+    },
+    "browserslist": [
```
Comment (MattIPv4) [inline ajax/libs/guidechimp/package.json]: Not needed

**Comment by r-brown** [comment]
Closing this PR as there requested changes do not comply with the established library CI and npmjs requirements.

**Comment by MattIPv4** [comment]
A package.json file here is not the same concept as a npmjs package.json file - these are completely different and one does not affect the other.

**Pull request closed without merging**